pkgs/development/python-modules/jaraco_collections/default.nix

```diff
@@ -22,9 +23,12 @@ buildPythonPackage rec {
   '';
 
   nativeBuildInputs = [
+    setuptools
     setuptools-scm
   ];
 
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
```

Comment:
```suggestion
  env.SETUPTOOLS_SCM_PRETEND_VERSION = version;
```

Reply:
Good catch, will be fixed by https://github.com/NixOS/nixpkgs/pull/241067.